ui/src/containers/DefaultErrorBoundary.jsx

```diff
@@ -1,63 +1,37 @@
 import React from 'react';
-import { bindActionCreators } from 'redux';
-import { connect } from 'react-redux';
-import { store } from '../store';
+import { ErrorBoundary } from 'react-error-boundary';
 
 import { logFrontEndTraceBack } from '../actions/beamline';
+import { Button } from 'react-bootstrap';
+import { useDispatch } from 'react-redux';
 
-class DefaultErrorBoundary extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = { error: null, errorInfo: null, store: null };
-  }
+function DefaultErrorBoundary(props) {
+  const { children } = props;
+  const dispatch = useDispatch();
 
-  componentDidCatch(error, errorInfo) {
-    this.setState({
-      error,
-      errorInfo,
-      store: store.getState(),
-    });
-    this.props.logFrontEndTraceBack(errorInfo.componentStack);
-  }
-
-  render() {
-    if (this.state.errorInfo) {
-      return (
-        <div>
-          <h3>Something went wrong</h3>
-          <p>
-            We are terribly sorry but something went wrong, the error has been
-            logged. If it remains please contact suport.
+  return (
+    <ErrorBoundary
+      fallbackRender={({ error }) => (
+        <div className="p-4">
+          <h3 className="mb-3">Something went wrong</h3>
+          <p className="mb-2">
+            We are terribly sorry but an error occurred:{' '}
+            <code>{error?.toString()}</code>
           </p>
           <p>
-            Simply, <a href="/datacollection">reloading</a> the page might fix
-            the issue.
+            Reloading the page might fix the issue. If it remains, please
+            contact support.
           </p>
-          <details style={{ whiteSpace: 'pre-wrap' }}>
-            {this.state.error && this.state.error.toString()}
-            <br />
-            {this.state.errorInfo.componentStack}
-            <br />
-            {JSON.stringify(this.state.store)}
```

Comment:
I'm no longer displaying the component stack and store snapshot. I still send both to the server via the `onError` callback and the `logFrontEndTraceBack` action, though, so they still get logged server-side.